amy/extrequests/templatetags/request_membership.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
+
+from django import template
+from django.utils.safestring import mark_safe
+
+from workshops.models import Membership
+
+register = template.Library()
+
+
+@register.simple_tag
+def membership_description(membership: Membership):
+    if not isinstance(membership, Membership):
```

Comment:
Is it expected for this tag to work on objects other than Membership?

If it's expected then the type hinting is not correct. Otherwise I'd remove this condition.